src/aspire/estimation/covar2d.py

```diff
@@ -9,25 +9,107 @@
 
 from aspire.utils.matrix import shrink_covar
 from aspire.utils.optimize import conj_grad
-from aspire.denoise.covar2d import RotCov2D
-from aspire.utils import ensure
 
+from aspire.utils import ensure
 
 logger = logging.getLogger(__name__)
 
 
-class Cov2DCTF(RotCov2D):
+class RotCov2D:
     """
-    Define a derived class for denoising 2D images using CTF information and the Covariance Wiener Filtering (CWF)
-    Cov2D method described in
+    Define a class for denoising 2D images using the Cov2D method with CTF information described in
```

Comment:
This needs to be updated. The class longer does denoising, but estimates the 2D covariance.

Reply:
done.